src/main/java/core/basesyntax/RandomNumberFigures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomNumberFigures {
+    public int randomInt(int number) {
```

Comment:
what about method naming?
```suggestion
    public int getRandomInt(int number) {
```

Reply:
I did it.